README.md

```diff
@@ -100,7 +112,8 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema organizes how database tables are constructed. Which columns are named what and contain which information.
+To use people and wishes as an example for how to represent a one-to-many relationship in a schema, the table for Person would have a few attributes like, name and age. The database would also give them a unique id. The table for Wishes would have it's own attributes like number of wish or what the wish was. We would need to include the Person's unique id as an attribute to each wish as well. So one Person can have many Wishes.
```

Comment:
perfect answer! We were looking for SQL here, I recommend you give it a try - happy to give feedback on this